api/v1alpha5/sysprep/sysprep.go

```diff
@@ -42,6 +44,21 @@ type Sysprep struct {
 
 	// UserData is a representation of the Sysprep UserData key.
 	UserData UserData `json:"userData"`
+
+	// +optional
+
+	// ExpirePasswordAfterNextLogin indicates whether or not the local Administrators
+	// group accounts are required to change their password after the next login.
+	ExpirePasswordAfterNextLogin bool `json:"expirePasswordAfterNextLogin,omitempty"`
```

Comment:
Please update the documentation to make the behavior here very clear, i.e. if this field is set to `true`, it will be set _back_ to `false` the first time we are able to customize the VM with the option to expire the password.

We should be very explicit when we are writing back to / mutating the spec from the controller.

Reply:
Resolving - this field will not be a latch